box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxArray.java

```diff
@@ -0,0 +1,43 @@
+package com.box.androidsdk.content.models;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * A collection that contains BoxJsonObject items
+ *
+ * @param <E> the type of elements in this partial collection.
+ */
+public class BoxArray<E extends BoxJsonObject> extends BoxList<E> {
```

Comment:
Since we want the BoxArray to just represent a simple flat list, let's implement Collection<E> (like BoxList) instead of extending from BoxList.